src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Cipher.java

```diff
@@ -641,15 +692,18 @@ private int implUpdate(byte[] in, int inOfs, int inLen,
             }
             if (inLen > 0) {
                 if (encrypt) {
-                    k += token.p11.C_EncryptUpdate(session.id(), 0, in, inOfs,
-                            inLen, 0, out, (outOfs + k), (outLen - k));
+                    k += token.p11.C_EncryptUpdate(session.id(),
+                            0, in, inOfs, inLen,
+                            0, out, (outOfs + k), (outLen - k));
```

Comment:
Is this an actual change?

Reply:
@theRealAph: no, it looked to me easier to read if we have input-related parameters separated from output ones. So I did this reordering in a separate commit (0a777e94229723376e1264e87cbf0ba805dc736f), to distinguish it from the core change (in c4419bfecc0b795f6a1bdcbe5c5a6aa0142350ea).

We can revert 0a777e94229723376e1264e87cbf0ba805dc736f if it confuses more than it improves.

Reply:
I prefer to not making changes like this.

Reply:
Reverted, see 68197517ec2f3e7e27debf6d2fca15a4b43c4e4e (whitespace-only change).

NOTE: new calls of `C_EncryptUpdate`/`C_DecryptUpdate`/`C_EncryptFinal`/`C_DecryptFinal` were retained in the re-arrangement style, as I found that we didn't introduce this re-arrangement, it was already present in most of the calls inside `P11Cipher::implUpdate()` and `P11Cipher::implDoFinal()`.